zavod/zavod/shed/fsf.py

```diff
@@ -50,8 +50,20 @@ def parse_address(context: Context, el: Element) -> Optional[Entity]:
     )
 
 
-def parse_sanctions(context: Context, entity: Entity, entry: Element) -> None:
+def parse_sanctions(
+    context: Context,
+    entity: Entity,
+    entry: Element,
+    program_key_override: str | None = None,
```

Comment:
can we just set the default value here to `programme`? I'd rename the variable to `program_attrib` (attributes in the XML sense), and then the implementation would just be `regulation.get(program_attrib)`.